### ISSUE TITLE
Wrong versioning of test in beta

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-om20#7]
This test is in beta and should be labeled with version <1.0. See guidance here:
https://github.com/opengeospatial/cite/wiki/ETS-Names-and-Versioning

**Comment by epicureanism** [comment]
Got it!
Just modified the version number and relnotes.
https://github.com/opengeospatial/ets-om20/commit/b45826c40566c89a0afd419550ba12b5068f0c1f #

**Comment by lgoltz** [comment]
Version was changed. 
The tags 1.0 and 1.1 already exists. The should be removed as soon 1.0 is official released (or 0.4 as I assume older pre versions must not be kept).

**Comment by epicureanism** [comment]
@lgoltz Not sure how to remove the tags or how to modify them (seems no permission to do so), any suggestions?

**Comment by dstenger** [comment]
We can remove (or rename) the wrong tags from repository. Can you please prepare the next release as usual and use the correct version number for that (0.x)?

**Comment by dstenger** [comment]
Version 1.2 has never been released. So, versions 1.2/0.3 and 0.4 were merged and 0.3 is the next release.

https://github.com/opengeospatial/ets-om20/commit/019bd44694005310eacb393624e98b5a9510fa75

**Comment by dstenger** [comment]
Also, tags 1.0 and 1.1 were renamed to 0.1 and 0.2.

```
git tag 0.1 1.0
git tag 0.2 1.1
git tag -d 1.0
git tag -d 1.1
git push origin --tags
git push origin -d 1.0
git push origin -d 1.1
```
